src/Shared/Traits.cs

```diff
@@ -109,6 +109,8 @@ internal class EscapeHatches
         /// </summary>
         public readonly bool AlwaysUseContentTimestamp = Environment.GetEnvironmentVariable("MSBUILDALWAYSCHECKCONTENTTIMESTAMP") == "1";
 
+        public readonly bool LogTaskInputsVerbose = Environment.GetEnvironmentVariable("MSBUILDLOGTASKINPUTSVERBOSE") == "1";
```

Comment:
Did this leak in from your other change?

Reply:
Whoops! I'll revert that.

Thanks!